nixos/doc/manual/from_md/release-notes/rl-2205.section.xml

```diff
@@ -1880,7 +1880,8 @@
         <para>
           <literal>services.xserver.desktopManager.xfce</literal> now
           includes Xfce’s screen locker,
-          <literal>xfce4-screensaver</literal>.
+          <literal>xfce4-screensaver</literal> that is enabled by
+          default.
```

Comment:
can you link to the option to disable it explicitely?

Reply:
I just added it in the documentation.